main.go

```diff
@@ -54,27 +61,38 @@ func actionMain(_ map[string]string, _ *ezactions.RunResources) (map[string]stri
 	oauthClient := oauth2.NewClient(ctx, tokenSource)
 	githubClient := github.NewClient(oauthClient)
 
-	for {
-		response, err := createFile(eventPullRequestTitle, eventPullRequestIssuer, pullRequestNumber, githubClient, ctx)
-		if response != nil && response.StatusCode == 422 {
-			log.Printf("file already exists for %v: %v, considering as success", pullRequestNumber, err)
-			return map[string]string{}, nil
-		}
-		if err != nil {
-			if _, ok := err.(*github.RateLimitError); ok {
-				waitDuration := time.Now().Sub(response.Rate.Reset.Time)
-				log.Printf("hit rate limit, will need to wait %v sec and retry", waitDuration.Seconds())
-				time.Sleep(waitDuration)
-			} else {
-				return nil, fmt.Errorf("failed to create file for %v: %v", pullRequestNumber, err)
-			}
+	pullRequestTitle := eventPullRequestTitle
+	pullRequestIssuer := eventPullRequestIssuer
+
+	if len(pullRequestTitle) == 0 {
+		pr, _, err := githubClient.PullRequests.Get(ctx, "apiiro", "lim", pullRequestIntNumber)
```

Comment:
this action is only used by `lim`?
if potentially this can be used by other repos as well, then maybe we should get `apiiro` and `lim` as external arguments

Reply:
Yeah its only used by lim.... But fine i can add it